view.go

```diff
@@ -87,6 +87,24 @@ type View struct {
 	// If Frame is true, a border will be drawn around the view.
 	Frame bool
 
+	// FrameColor allow to configure the color of the Frame when it is not highlighted.
+	FrameColor Attribute
+
+	// FrameRunes allows to define custom runes for the frame edges.
+	// The rune slice can be defined with 3 different lengths.
+	// If slice doesn't match these lengths, default runes will be used instead of missing one.
+	//
+	// 2 runes with only horizontal and vertical edges.
+	//  []rune{'─', '│'}
+	//  []rune{'═','║'}
+	// 6 runes with horizontal, vertical edges and top-left, top-right, bottom-left, bottom-right cornes.
+	//  []rune{'─', '│', '┌', '┐', '└', '┘'}
+	//  []rune{'═','║','╔','╗','╚','╝'}
+	// 11 runes which can be used with `gocui.Gui.SupportOverlaps` property.
+	//  []rune{'─', '│', '┌', '┐', '└', '┘', '├', '┤', '┬', '┴', '┼'}
+	//  []rune{'═','║','╔','╗','╚','╝','╠','╣','╦','╩','╬'}
+	FrameRunes []rune
```

Comment:
Maybe we should somewhere in the code check but i'm not sure about that, what do you think?

Reply:
I'm not sure how exactly it needs to be checked. There are some checks that are used for certain runes if it has length which is required for that. But I'm not sure if it needs to be more limited or specified, or if that is enough.

I meant these:
https://github.com/awesome-gocui/gocui/blob/cb1cfb0a01a11dfabedad1ce37efac3b3f3f121a/gui.go#L593
https://github.com/awesome-gocui/gocui/blob/cb1cfb0a01a11dfabedad1ce37efac3b3f3f121a/gui.go#L710
https://github.com/awesome-gocui/gocui/blob/cb1cfb0a01a11dfabedad1ce37efac3b3f3f121a/gui.go#L689

Reply:
But if you can see specific case where it might be problem, let me know. I couldn't think of any for now, but I might be missing it :)

Reply:
Your documentation about this function explains the lengths very well so this is probably already good enough